spec/unit/util/package/version/range_spec.rb

```diff
@@ -48,13 +48,13 @@ def <=>(other)
     end
 
     it 'it includes specified version' do
-      vr = Puppet::Util::Package::Version::Range.parse('>=3', IntegerVersion)
+      vr = Puppet::Util::Package::Version::Range.parse('>= 3', IntegerVersion)
```

Comment:
space is not mandatory, you could keep existing tests as they are when possible and add few new tests for the new cases

Reply:
Done. Let me know if they are now as expected.